### ISSUE TITLE
Corpus of acceptance test cases

### ISSUE THREAD

**Issue by dr-jts** [TWKB/Specification#25]
It would be useful to have a standard corpus of examples of TWKB-encoded geometry.  This could be used as acceptance tests for implementors.

**Comment by nicklasaven** [comment]
Yes, sounds good. There are some examples in the regress tests of PostGIS. But we could definitely put it in the spec (encoded as base64 as of #24)
